tests/dialects/test_snowflake.py

```diff
@@ -1443,11 +1442,29 @@ def test_show(self):
         )
 
         ast = parse_one('SHOW PRIMARY KEYS IN "TEST"."PUBLIC"."customers"', read="snowflake")
+        self.assertIsInstance(ast, exp.Show)
         table = ast.find(exp.Table)
 
         self.assertIsNotNone(table)
         self.assertEqual(table.sql(dialect="snowflake"), '"TEST"."PUBLIC"."customers"')
 
+        self.validate_identity("SHOW COLUMNS")
+        self.validate_identity("SHOW COLUMNS IN TABLE dt_test")
+        self.validate_identity("SHOW COLUMNS LIKE '_foo%' IN TABLE dt_test")
+        self.validate_identity("SHOW COLUMNS IN VIEW")
+        self.validate_identity("SHOW COLUMNS LIKE '_foo%' IN VIEW dt_test")
+
+        ast = parse_one("SHOW COLUMNS LIKE '_testing%' IN dt_test", read="snowflake")
+        self.assertIsInstance(ast, exp.Show)
```

Comment:
assert instance tests are also not really necessary because if they were commands, the ast.finds would fail, so we can get rid of those